libs/core/src/lib/token/tokenizer.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="fd-tokenizer" (focus)="tokenizerHasFocus = true" (blur)="tokenizerHasFocus = false; null"
```

Comment:
`(blur)="tokenizerHasFocus = false; null"` I think that `null` is a leftover from some previous implementation